src/lightbox-overlay.component.ts

```diff
@@ -42,6 +41,10 @@ export class LightboxOverlayComponent implements AfterViewInit, OnDestroy {
     this._lightboxEvent.broadcastLightboxEvent({ id: LIGHTBOX_EVENT.CLOSE, data: null });
   }
 
+  ngOnInit() {
+    this._documentRef = window.document;
```

Comment:
can this be inside the constructor instead of putting inside `ngOnInit` ?

Reply:
You're right. This is `constructor` logic.